### PR TITLE
Modify the user profile path

### PR DESCRIPTION
**Pull request by baras**
#15

### CODE REVIEW

File: Handler/User.hs [hunk 1]
```diff
@@ -11,7 +11,7 @@ import           Utils.LocalTasks
 getProfileR :: Handler Html
 getProfileR = do
     (userId, user) <- requireAuthPair
-    getUserR (userIdent user)
+    redirect $ UserR (userIdent user)
```
Comment (amitaibu) [inline Handler/User.hs:14]: `UserR userId` will probably also work.

Comment (amitaibu) [reply Handler/User.hs:14]:
``` haskell
(_, user) <- requireAuthPair
redirect $ UserR (userIdent user)
```

**Comment by baras** [comment]
@amitaibu Locally everything works as expected. I will look into fixing the tests.

**Comment by baras** [comment]
anim

**Comment by baras** [comment]
@amitaibu Ready for review.
Should the menu item and breadcrumbs be changed to the new path?

**Comment by amitaibu** [comment]
Merged, thanks.

> Should the menu item and breadcrumbs be changed to the new path?

Your call - lets do what makes sense

**Pull request merged**